src/main/scala/metagenomica/loquats/5.assignment.scala

```diff
@@ -79,6 +79,10 @@ extends DataProcessingBundle(
     val lcaWriter = CSVWriter.open(lcaFile.toJava , append = true)
     val bbhWriter = CSVWriter.open(bbhFile.toJava , append = true)
 
+    // writing headers first:
+    lcaWriter.writeRow(List("Read-ID", "LCA-node"))
+    bbhWriter.writeRow(List("Read-ID", "BBH-node"))
```

Comment:
@marina-manrique take a look please, are these csv-headers clear enough? This is for assignment output.